vmdb/tools/fix_auth/rails_mock.rb

```diff
@@ -0,0 +1,11 @@
+unless defined?(Rails)
+  module Rails
+    def self.rails_root
+      @rails_root ||= ENV["RAILS_ROOT"]
+    end
+
+    def self.env
+      @env ||= ActiveSupport::StringInquirer.new("production")
```

Comment:
agreed. I like 16 better.
But if we didn't want to modify rails, this is an alternative.